res/verify_response_page_template_vector_im

```diff
@@ -1,49 +1,49 @@
-<!DOCTYPE html>
-<html>
-<head>
-<meta charset="utf-8" />
-<title></title>
-<style>
-
-body {
-    font-family: 'Open Sans', Helvetica, Arial, Sans-Serif;
-    font-color: #454545;
-    font-size: 12pt;
-    margin: 1em;
-}
-#vectorlogo {
-    margin: auto;
-    text-align: center;
-    padding: 10px;
-}
-#message {
-    width: 300px;
-    text-align: left;
-    padding: 1em;
-    margin-bottom: 40px;
-    margin-left: auto;
-    margin-right: auto;
-    margin-top: 50px;
-
-    -webkit-border-radius: 10px;
-    -moz-border-radius: 10px;
-    border-radius: 10px;
-
-    -webkit-box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.15);
-    -moz-box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.15);
-    box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.15);
-
-    background-color: #f8f8f8;
-    border: 1px #ccc solid;
-}
-</style>
-</head>
-<body>
-<div id="vectorlogo">
-<img src="//riot.im/img/external/riot-logo.svg" alt="riot.im" width="256" height="256" />
-</div>
-<div id="message">
-<p>%(message)s</p>
-</div>
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title></title>
+<style>
+
+body {
+    font-family: 'Open Sans', Helvetica, Arial, Sans-Serif;
+    font-color: #454545;
+    font-size: 12pt;
+    margin: 1em;
+}
+#vectorlogo {
+    margin: auto;
+    text-align: center;
+    padding: 10px;
+}
+#message {
+    width: 300px;
+    text-align: left;
+    padding: 1em;
+    margin-bottom: 40px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 50px;
+
+    -webkit-border-radius: 10px;
+    -moz-border-radius: 10px;
+    border-radius: 10px;
+
+    -webkit-box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.15);
+    -moz-box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.15);
+    box-shadow: 0px 0px 20px 0px rgba(0,0,0,0.15);
+
+    background-color: #f8f8f8;
+    border: 1px #ccc solid;
+}
+</style>
+</head>
+<body>
+<div id="vectorlogo">
+<img src="//static.element.io/images/email-logo.png" alt="riot.im" width="256" height="256" />
```

Comment:
```suggestion
<img src="//static.element.io/images/email-logo.png" alt="element.io" width="256" height="256" />
```

Reply:
Or maybe just "Element"?

Reply:
Thanks! And yeah, we want to emphasize the brand name instead of the domain name (which we had to do in the past for legal reasons) :)